admin/class-admin-apple-bulk-export-page.php

```diff
@@ -83,7 +83,7 @@ public function set_title( $admin_title ) {
 	 * @access public
 	 */
 	public function build_page() {
-		$ids = isset( $_GET['ids'] ) ? sanitize_text_field( wp_unslash( $_GET['ids'] ) ) : null;
+		$ids = isset( $_GET['ids'] ) ? sanitize_text_field( wp_unslash( $_GET['ids'] ) ) : null; // input var ok.
```

Comment:
What is the specific error being thrown here? I'd rather fix the code to be compliant with how the rule wants it to be written than to just ignore it.

Reply:
All superglobals need a `// input var ok` comment to indicate you have manually inspected them. There's no error to correct.

Reply:
Good point. However, I'd like to use the new format for whitelisting, which is in use elsewhere in the codebase:

```php
// phpcs:ignore WordPress.VIP.SuperGlobalInputUsage.AccessDetected
```